extensions/tokenizers/src/main/java/ai/djl/huggingface/tokenizers/HuggingFaceTokenizer.java

```diff
@@ -303,86 +338,118 @@ public String decode(long[] ids) {
+     * @param enabled whether to enable default truncation behavior
      */
-    public void setTruncationAndPadding(
-            TruncationStrategy truncationStrategy,
-            PaddingStrategy paddingStrategy,
-            long maxLength) {
-        setTruncationAndPadding(truncationStrategy, paddingStrategy, maxLength, 0, 0);
+    public void setTruncation(boolean enabled) {
+        TruncationStrategy strategy =
+                enabled ? TruncationStrategy.LONGEST_FIRST : TruncationStrategy.DO_NOT_TRUNCATE;
+        if (strategy != truncation) {
+            truncation = strategy;
+            updateTruncationAndPadding();
+        }
     }
 
     /**
-     * Sets the truncation and padding behavior for the tokenizer.
+     * Enables truncation to specified length for the tokenizer.
      *
-     * @param truncationStrategy the {@code TruncationStrategy} to use
-     * @param paddingStrategy the {@code PaddingStrategy} to use
-     * @param maxLength the maximum length to pad/truncate sequences to
-     * @param stride value to use when handling overflow
+     * @param maxLength truncate to specified length
      */
-    public void setTruncationAndPadding(
-            TruncationStrategy truncationStrategy,
-            PaddingStrategy paddingStrategy,
-            long maxLength,
-            long stride) {
-        setTruncationAndPadding(truncationStrategy, paddingStrategy, maxLength, stride, 0);
+    public void setTruncation(int maxLength) {
+        truncation = TruncationStrategy.LONGEST_FIRST;
+        this.maxLength = maxLength;
+        updateTruncationAndPadding();
     }
 
-    /**
-     * Sets the truncation and padding behavior for the tokenizer.
-     *
-     * @param truncationStrategy the {@code TruncationStrategy} to use
-     * @param paddingStrategy the {@code PaddingStrategy} to use
-     * @param maxLength the maximum length to pad/truncate sequences to
-     * @param stride value to use when handling overflow
-     * @param padToMultipleOf pad sequence length to multiple of value
-     */
-    public void setTruncationAndPadding(
-            TruncationStrategy truncationStrategy,
-            PaddingStrategy paddingStrategy,
-            long maxLength,
-            long stride,
-            long padToMultipleOf) {
-        setTruncation(truncationStrategy, maxLength, stride);
-        setPadding(paddingStrategy, maxLength, padToMultipleOf);
+    /** Enables truncation to only truncate the first item. */
+    public void setTruncateFirstOnly() {
+        if (truncation != TruncationStrategy.ONLY_FIRST) {
+            truncation = TruncationStrategy.ONLY_FIRST;
+            updateTruncationAndPadding();
+        }
+    }
+
+    /** Enables truncation to only truncate the second item. */
+    public void setTruncateSecondOnly() {
+        if (truncation != TruncationStrategy.ONLY_SECOND) {
+            truncation = TruncationStrategy.ONLY_SECOND;
+            updateTruncationAndPadding();
+        }
```

Comment:
Should we also provide `maxLength` as an argument? Right now there doesn't seem like an way to use these strategies with custom lengths via the API.